src/packages/@ncigdc/components/Header.js

```diff
@@ -148,6 +142,12 @@ const Header = compose(
               </ExternalLink>
 
             </li>}
+          <li>
```

Comment:
You need to jump one more link, thats the old analysis link. Maybe we should delete it?